### PR TITLE
v1: enable filtering by subreddit on messages endpoint

### PR DESCRIPTION
**Pull request by curioussavage**
:eyeglasses: @ajacksified

### CODE REVIEW

File: old-src/endpoints/v1.es6.js [hunk 1]
```diff
@@ -1239,7 +1239,10 @@ class APIv1Endpoint {
   get messages () {
     return bindAll({
       buildOptions: function(options) {
-        const uri = options.origin + '/message/' + (options.view || 'inbox');
+        const { filter, subredditName, origin, view } = options;
```
Comment (ajacksified) [inline old-src/endpoints/v1.es6.js]: you can default `view` here:

`const { filter, subredditName, origin, view = 'inbox' } = options;`

**Comment by ajacksified** [comment]
minor nit, :nail_care:

**Comment by curioussavage** [comment]
:haircut:

**Comment by ajacksified** [comment]
:fish:

**Pull request closed without merging**